ivy/functional/ivy/manipulation.py

```diff
@@ -1146,6 +1146,86 @@ def tile(
     -------
     ret
         The tiled output array.
+    
+
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` input:
+    
+    >>> x = ivy.array([1,2,3,4])
+    >>> y = ivy.tile(x,(3,))
+    >>> print(y)
+    [1,2,3,4,1,2,3,4,1,2,3,4]
+
+    >>> x = ivy.array([[1,2,3], \
+                       [4,5,6]])
+    >>> y = ivy.tile(x, (2,3))
+    >>> print(y)
+    {
+        [[1,2,3,1,2,3,1,2,3]
+         [4,5,6,4,5,6,4,5,6]
+         [1,2,3,1,2,3,1,2,3]
+         [4,5,6,4,5,6,4,5,6]]
+    }
+
+    With :code:`ivy.NativeArray` input:
+    
+    >>> x = ivy.native_array([[[0], \
+                               [1]]])
+    >>> y = ivy.tile(x,(2,2,3))
+    >>> print(y)
+    {
+        [[[0,0,0],
+          [1,1,1],
+          [0,0,0],
+          [1,1,1]],
+         [[0,0,0],
+          [1,1,1],
+          [0,0,0],
+          [1,1,1]]
+        ]
+    }
+
+    With :code:`ivy.Container` input:
+    
+    >>> x = ivy.Container( a = ivy.array([0,1,2]), b = ivy.array([[3],[4]]))
+    >>> y = ivy.tile(x, (2))
+    >>> print(y)
+    {
+        a = [0,1,2,0,1,2]
+        b = [[3,3],
+             [4,4]]
+    }
+
+    Instance Method Examples
```

Comment:
The instance methods example belong to the `tile` instances in the `ivy/array` and `ivy/container`.